src/components/AddressInput/AddressInput.spec.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
-import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
-import {addressInputDriverFactory} from './AddressInput.driver';
-import {AddressInput} from '.';
-import {mount} from 'enzyme';
-import {GoogleMapsClientStub} from 'wix-ui-core/dist/src/components/AddressInput/GoogleMapsClientStub';
-import * as helper from 'wix-ui-core/dist/src/components/AddressInput/AddressInputTestHelper';
+import { createDriverFactory } from 'wix-ui-test-utils/driver-factory';
```

Comment:
Why the lint changes suddenly?
This should not happen until we migrate to yoshi 3 lint rules, and then we need 1 PR that fixs ALL lint

Reply:
This is prettier breaking lines and applying additional lint rules. If you run yoshi lint on this - it will not complain at all. remember that we want to get rid of wix-ui-backoffice anyway so I dont see any problem with this applying automatic lint while I change here something.